src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/GerritManagement/newServer.jelly

```diff
@@ -28,12 +28,6 @@ THE SOFTWARE.
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout">
   <l:layout norefresh="true" permission="${app.ADMINISTER}" title="${%Add New Server}">
-    <l:side-panel>
```

Comment:
Why remove the side panel? It's nothing important, just curious.

Reply:
Following the the design library recommendations (https://weekly.ci.jenkins.io/design-library/Layouts/) hierarchical links should be avoided